src/components/Form/Form.js

```diff
@@ -1,83 +1,104 @@
+        const {id} = event.target; //help to add value of attribute from input
+
+        this.setState({
+            [id]: event.target.value
+        });
     };
-  }
 
-  static getDerivedStateFromProps(nextProps, prevState) {
-    if (nextProps.post.id !== prevState.id) {
-      return {
-        id: nextProps.post.id,
-        text: nextProps.post.text,
-        title: nextProps.post.title
-      };
-    }
-    return null;
-  }
+    onReset = () => {
+        // todo: имплементнуть функцию скидывания значений, введеных пользователем
+        this.setState({ // зануляем значения в стейте, чтоб сбросить формочку
+            id: uniqId(),
+            titleInput: '',
+            text: ''
+        });
+    };
+
+    onSubmit = () => {
+        const {user, addPost} = this.props;
 
-  onLabelChange = e => {
-    const { id } = e.target;
-    this.setState({
-      [id]: e.target.value
-    });
-  };
+        const newPost = {
+            ...this.state,
+            data: new Date(),
+            mood: 'happy',
+            authorName: `${user.name} ${user.lastName}`
+        };
 
-  onSubmit = () => {
-    const { user, addPost } = this.props;
-    const newPost = {
-      ...this.state,
-      data: new Date(),
-      mood: 'happy',
-      authorName: `${user.name} ${user.lastName}`
+        addPost(newPost); // создаем новый пост
+
+        this.setState({ // зануляем значения в стейте, чтоб сбросить формочку
+            id: uniqId(),
+            titleInput: '',
+            text: ''
+        });
     };
 
-    addPost(newPost); // создаем новый пост
+    render() {
+        return (
+            <div className={CN}>
+                <h2>Create new Post</h2>
+                <div className="form-group">
+                    <label for="titleInput" className="input-group-text">Enter post title:</label>
+                    <input
+                        className="form-control"
+                        type="text"
+                        id="titleInput"
+                        onChange={this.onLabelChange}
+                        value={this.state.titleInput}
+                    />
+                </div>
+
+                <div className="form-group">
+                    <label for="text" className="input-group-text">Enter post text:</label>
+                    <input
+                        className="form-control"
+                        name="text-input"
+                        type="text"
+                        id="text"
+                        onChange={this.onLabelChange}
+                        value={this.state.text}
+                    />
+                </div>
+
+                <Button
+                    className={`btn-outline-secondary ${!this.state.titleInput.trim() && !this.state.text.trim() && 'myTooltip'}`}
+                    onClick={this.onSubmit}
+                    label="Add post"
+                    isDisable={!this.state.text.trim() && !this.state.titleInput.trim()}
```

Comment:
You can use destructing values from state
`const {text, titleInput} = this.state;`
in the begining of render method

Reply:
and then use it in all places in render